.gitignore

```diff
@@ -1 +1,2 @@
 .vscode/
+build/*
```

Comment:
Do we need the trailing `*`?  .gitignore is supposed to treat `build/` to mean ignore all files in this directory and below.

Do we need this line at all?  Probably best to build completely outside the source tree.

Reply:
The trailing `*` is not needed.

While there's certainly some advantages to having a read only source tree, it's also convenient to have the option to build in-tree under a `build/` directory.